
# [CVE-2020-15141](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-15141)

## Description

- `In openapi-python-client before version 0.5.3, there is a path traversal vulnerability. If a user generated a client using a maliciously crafted OpenAPI document, it is possible for generated files to be placed in arbitrary locations on disk.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 4.0

## Scores

- **Exploitability Score**:
  - 8.0
- **Impact Score**:
  - 2.9
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2020-08-14 17:15:13
- **Vulnerability Status**:
  - Analyzed

## References

- **security-advisories@github.com**: https://github.com/triaxtec/openapi-python-client/blob/main/CHANGELOG.md#053---2020-08-13
- **security-advisories@github.com**: https://github.com/triaxtec/openapi-python-client/commit/3e7dfae5d0b3685abf1ede1bc6c086a116ac4746
- **security-advisories@github.com**: https://github.com/triaxtec/openapi-python-client/security/advisories/GHSA-7wgr-7666-7pwj
- **security-advisories@github.com**: https://pypi.org/project/openapi-python-client
